doc/source/whatsnew/v1.2.0.rst

```diff
@@ -21,7 +21,8 @@ to pass a dictionary of parameters to the storage backend. This allows, for
 example, for passing credentials to S3 and GCS storage. The details of what
 parameters can be passed to which backends can be found in the documentation
 of the individual storage backends (detailed from the fsspec docs for
-`builtin implementations`_ and linked to `external ones`_).
+`builtin implementations`_ and linked to `external ones`_). See
```

Comment:
are these referenced in io.rst (more important that they are there), ok if they are here *as well* (but not really necessary)

Reply:
I phrased it a bit differently (one general link, one specific instead of two specific) - I'll make the two places more similar.